doc/index.rst

```diff
@@ -27,6 +27,8 @@ NixOps
    release-notes/1.x
 
 
+.. nixops_plugins_doc::
```

Comment:
What is the purpose of the directive?

Reply:
To dynamically (at build time) include docs from all plugins in the closure.

Reply:
It's possible that there is a better way to accomplish this. I'm not really too familiar with Sphinx.